modules/synthetic-protocol/src/lib.rs

```diff
@@ -179,7 +179,7 @@ impl<T: Trait> Module<T> {
 		);
 
 		ensure!(
-			T::LiquidityPools::is_allowed_position(pool_id, currency_id, Leverage::LongOne),
+			T::LiquidityPools::is_allowed_position(pool_id, currency_id, Leverage::LongTwo),
```

Comment:
this isn't right, but you can fix it in next PR by change this to use `can_mint`

Reply:
yes that's the plan